src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  padding-top: 2px;
+  padding-left: 2px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 15px;
+  height: 14px;
```

Comment:
Size star on design. You need to fix it
<img width="958" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/bed653a7-d503-431a-af7f-771780451fdf">